src/api/app/assets/javascripts/webui/notification_toolbar.js

```diff
@@ -0,0 +1,7 @@
+$(window).scroll(function () {
+    if (this.scrollY > 100) {
+        $('.notification-controls').addClass('sticking');
```

Comment:
@OleksandrOrlov is this a left over, I cannot find the `notification-controls` class in the html code

Reply:
You're right. This is not needed here anymore. Thanks for the finding. Sorry:)